### PR TITLE
pdctl: add cluster command

### PR DESCRIPTION
**Pull request by nolouch**
PTAL @disksing @siddontang

### CODE REVIEW

**Comment by siddontang** [comment]
PTAL @disksing

File: pdctl/command/cluster_command.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	"github.com/spf13/cobra"
+)
+
+const clusterPrefix = "pd/api/v1/cluster"
+
+// NewClusterCommand return a cluster subcommand of rootCmd
+func NewClusterCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "cluster",
+		Short: "show the cluster information",
```
Comment (andelf) [inline pdctl/command/cluster_command.go]: ~~the~~

File: pdctl/command/cluster_command.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		Run:   showClusterCommandFunc,
+	}
+	return cmd
+}
+
+func showClusterCommandFunc(cmd *cobra.Command, args []string) {
+	r, err := doRequest(cmd, clusterPrefix, http.MethodGet)
+	if err != nil {
+		fmt.Printf("Failed to get the cluster information: %s", err)
```
Comment (andelf) [inline pdctl/command/cluster_command.go]: ~~the~~

**Comment by disksing** [comment]
LGTM.

**Pull request merged**